components/eam/src/chemistry/pp_chemuci_linozv3_mam5_vbs/mo_imp_sol.F90

```diff
@@ -527,8 +527,10 @@ subroutine imp_sol( base_sol, reaction_rates, het_rates, extfrc, delt, &
                 !-----------------------------------------------------------------------
                 fail_cnt = fail_cnt + 1
                 nstep = get_nstep()
-                write(iulog,'('' imp_sol: Time step '',1p,e21.13,'' failed to converge @ (lchnk,lev,col,nstep) = '',4i6)') &
-                     dt,lchnk,lev,i,nstep
+                ! to reduce the unnecessary imp_sol printing in production runs
```

Comment:
```suggestion
                ! reduce the frequency of warning printing
```

Reply:
Once this PR is merged, all simulations will print these warnings at a lower frequency, including production runs.